prov/efa/src/rdm/rxr_ep.c

```diff
@@ -295,7 +295,7 @@ int rxr_ep_post_internal_rx_pkt(struct rxr_ep *ep, uint64_t flags, enum rxr_lowe
 
 	rx_pkt_entry->x_entry = NULL;
 
-	msg_iov.iov_base = (void *)rxr_pkt_start(rx_pkt_entry);
+	msg_iov.iov_base = (void *)rxr_pkt_entry_wiredata_start(rx_pkt_entry);
```

Comment:
Why not just `rx_pkt_entry->wiredata`?

Reply:
Thank you. Given your feedback, I have removed `rxr_pkt_start` and `rxr_pkt_end` entirely.